tests/reboot/reboot_test.go

```diff
@@ -104,6 +105,71 @@ var _ = Describe("{RebootOneNode}", func() {
 	})
 })
 
+var _ = Describe("{ReallocateSharedMount}", func() {
+	var contexts []*scheduler.Context
+
+	It("has to schedule apps and reboot node(s) with shared volume mounts", func() {
+		//var err error
+		contexts = make([]*scheduler.Context, 0)
+
+		for i := 0; i < Inst().ScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("reallocate-mount-%d", i))...)
+		}
+
+		ValidateApplications(contexts)
+
+		Step("get nodes with shared mount and reboot them", func() {
+			for _, ctx := range contexts {
+				vols, err := Inst().S.GetVolumes(ctx)
+				Expect(err).NotTo(HaveOccurred())
+				for _, vol := range vols {
+					if vol.Shared {
+						n, err := Inst().V.GetNodeForVolume(vol, 1*time.Minute, 5*time.Second)
+						Expect(err).NotTo(HaveOccurred())
+						logrus.Infof("volume %s is attached on node %s [%s]", vol.ID, n.SchedulerNodeName, n.Addresses[0])
+						StopVolDriverAndWait([]node.Node{*n})
+						err = Inst().V.StopDriver([]node.Node{*n}, false, nil)
+						Expect(err).NotTo(HaveOccurred())
+						err = Inst().N.RebootNode(*n, node.RebootNodeOpts{
+							Force: true,
+							ConnectionOpts: node.ConnectionOpts{
+								Timeout:         1 * time.Minute,
+								TimeBeforeRetry: 5 * time.Second,
+							},
+						})
+						Expect(err).NotTo(HaveOccurred())
+						logrus.Infof("waiting for 20 seconds to node %s to reboot", n.SchedulerNodeName)
+						time.Sleep(20 * time.Second)
```

Comment:
we can use err = Inst().N.TestConnection for check if node is back up instead of sleep